tirocks/src/lib.rs

```diff
@@ -1,8 +1,9 @@
-#[cfg(test)]
-mod tests {
-    #[test]
-    fn it_works() {
-        let result = 2 + 2;
-        assert_eq!(result, 4);
-    }
-}
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+#![cfg_attr(feature = "nightly", features(io_error_more))]
```

Comment:
Is this a typo? What is the difference between features and feature?
```suggestion
#![cfg_attr(feature = "nightly", feature(io_error_more))]
```